examples/LexStatusCodes.rb

```diff
@@ -42,6 +42,11 @@ module LexStatusCodes
 
   LA_RELEASE_NO_UPDATE_AVAILABLE = 31
 
+  # CODE: LA_RELEASE_UPDATE_AVAILABLE_NOT_ALLOWED
+  # MESSAGE: The update available is not allowed for this license.
+
+  LA_RELEASE_UPDATE_AVAILABLE_NOT_ALLOWED = 32,
```

Comment:
@muneebkq This error code would be possible via CheckReleaseUpdate() only and that is not included yet.